src/core/qgsfileutils.h

```diff
@@ -87,6 +87,20 @@ class CORE_EXPORT QgsFileUtils
      * \since QGIS 3.2
      */
     static QString findClosestExistingPath( const QString &path );
+
+    /**
+     * Will check folder in an outward spiral up to 4 levels to check if the files exists
+     * found files will be highlighted in green of approval, otherwise in red.
```

Comment:
This method does not colorize anything, does it?

Reply:
Yes it does, does that prove to be a problem?

Reply:
Ah no sorry, this was supposed to be in the doc for the bad layer handler, not this. My bad.